profile/README.md

```diff
@@ -5,19 +5,6 @@ Espressif supports a large variety of open-source projects, including SDKs, comp
 All of Espressif’s official software, relating to the various series of ESP SoCs including ESP32  and ESP8266, are available on this GitHub site.
 To check out all the series of SoCs from Espressif, please visit our [ESP Product Selector](https://products.espressif.com).
 
-## Espressif Developers Conference 2023 - DevCon23
-<img width="1033" alt="LinkedIn_event" src="https://github.com/VojtechBartoska/.github/assets/76958047/8d4b9b47-f622-4cb0-a2ad-b7331808ed5f">
-
----
-
-We would like to invite you to our second edition of Espressif Developer Conference which will take place on September 12-13, 2023.
-
-Conference is for free and will be streamed on our [Espressif Youtube channel](https://www.youtube.com/@EspressifSystems).
-
-**You can look forward for various IoT topics, more details with full program is available at https://devcon.espressif.com/.**
-
----
-
 <details>
   <summary>For more information about Espressif organization and it's projects click here.</summary>
```

Comment:
@pedrominatel you need to remove also this line to have other content visible by default.

Reply:
Ok, tks!